cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -54,7 +53,7 @@
     {% endif %}
         {% if has_authors %}
             <span class="byline">
-            {%- for author in page_authors -%}
+            {%- for author in page.authors.order_by('name') -%}
                 {{- 'By ' if loop.index == 1 else ' and ' }}
```

Comment:
Is there a reason not to use `page.get_authors()` here, so we're not duplicating functionality?